browser/ui/webui/brave_rewards_ui.cc

```diff
@@ -534,22 +539,25 @@ void RewardsDOMHandler::GetReconcileStamp(const base::ListValue* args) {
 }
 
 void RewardsDOMHandler::OnGetAddresses(
+    const std::string func_name,
     const std::map<std::string, std::string>& addresses) {
-  if (web_ui()->CanCallJavascript()) {
+  if (web_ui()->CanCallJavascript() && !func_name.empty()) {
     base::DictionaryValue data;
     data.SetString("BAT", addresses.at("BAT"));
     data.SetString("BTC", addresses.at("BTC"));
     data.SetString("ETH", addresses.at("ETH"));
     data.SetString("LTC", addresses.at("LTC"));
 
-    web_ui()->CallJavascriptFunctionUnsafe("brave_rewards.addresses", data);
+    web_ui()->CallJavascriptFunctionUnsafe("brave_rewards." + func_name, data);
```

Comment:
We should prob check `func_name` through a whitelist to prevent arbitrary invoke.

Reply:
done